usr/share/rear/backup/BORG/default/800_prune_old_backups.sh

```diff
@@ -5,9 +5,9 @@
 
 if [ ! -z $BORGBACKUP_OPT_PRUNE ]; then
     # Purge old archives according user settings.
-    Log "Purging old Borg archives in repository $BORGBACKUP_REPO"
+    LogPrint "Purging old Borg archives in repository $BORGBACKUP_REPO"
 
-    borg prune --verbose --list ${BORGBACKUP_OPT_PRUNE[@]} \
```

Comment:
I'm no so sure why _--verbose_ option was used here as `borg prune` does not have _--verbose_ option ...